lib/trader-sdk/test/indicators/bb-percent-b-mfi.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { describe } from 'mocha';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dills122/trader-tools/pullRequest?prid=8102010)
```suggestion
import { describe } from "mocha";
```